index.html

```diff
@@ -76,136 +76,160 @@ <h1 id="my-h1">I’m Crispin<br/> Glad to see you!</h1>
                 </li>
             </ul>
         </section><br/>
+        <div class="all3-sections">
```

Comment:
- [ ] It is required that you remove the whole project section and add it dynamically using javascript. You will add the data in an array and iterate it through in the markup tags. You can use the popup array since the data are the same.